content/docs/guides/crosswalk/_index.md

```diff
@@ -1,4 +1,4 @@
 ---
 redirect_to: /docs/guides/crosswalk/aws/
 aliases: ["/docs/reference/crosswalk/"]
----
\ No newline at end of file
+---
```

Comment:
Should this be just `---`?

Reply:
Looks like https://www.pulumi.com/guides/crosswalk now returns a 404. I assume this wasn't intended?

Reply:
I think it's supposed to be https://www.pulumi.com/docs/guides/crosswalk

Reply:
I'll review this file in the other PR that hasn't been merged yet.